src/cromshell/utilities/IOUtils.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+
+from termcolor import colored
+
+
+def copy_file(source, destination):
+    source_path = Path(source)
+    destination_path = Path(destination)
+    destination_path.write_bytes(source_path.read_bytes())
+
+
+def turtle_dead(show_logo):
```

Comment:
It seems a bit weird to pass a boolean into these to decide if they do anything or not.  Maybe they should be guarded by a different function instead.  like log.turtle(turtleDead) and the logger can decide if it outputs turtle level log messages.